src/shared_modules/utils/tests/stringHelper_test.cpp

```diff
@@ -640,3 +660,84 @@ TEST_F(StringUtilsTest, splitToNumbers)
 
     EXPECT_ANY_THROW({ ret = Utils::splitToNumbers("1.1.1", ' '); });
 }
+
+#if __cplusplus >= 201703L
+
+TEST_F(StringUtilsTest, SplitEmptyStringSV)
+{
+    auto splitTextVector {Utils::splitView(""sv, '.')};
+    EXPECT_EQ(0ull, splitTextVector.size());
+}
+
+TEST_F(StringUtilsTest, SplitDelimiterNoCoincidenceSV)
+{
+    const auto splitTextVector {Utils::splitView("hello_world"sv, '.')};
+    EXPECT_EQ(1ull, splitTextVector.size());
+}
+
+TEST_F(StringUtilsTest, SplitDelimiterCoincidenceSV)
+{
+    const auto splitTextVector {Utils::splitView("hello.world"sv, '.')};
+    EXPECT_EQ(2ull, splitTextVector.size());
+    EXPECT_EQ(splitTextVector[0], "hello");
+    EXPECT_EQ(splitTextVector[1], "world");
+}
+
+TEST_F(StringUtilsTest, SplitDelimiterAtTheBeginningCoincidenceSV)
+{
+    const auto splitTextVector {Utils::splitView(".hello.world"sv, '.')};
+    EXPECT_EQ(2ull, splitTextVector.size());
+    EXPECT_EQ(splitTextVector[0], "hello");
+    EXPECT_EQ(splitTextVector[1], "world");
+}
+
+TEST_F(StringUtilsTest, SplitDelimiterAtTheEndCoincidenceSV)
+{
+    const auto splitTextVector {Utils::splitView("hello.world."sv, '.')};
+    EXPECT_EQ(2ull, splitTextVector.size());
```

Comment:
We expect size 3.
splitTextVector[0] = "hello"
splitTextVector[1] = "world"
splitTextVector[2] = ""

Reply:
Reverted